aws/buildspecs/sandbox/deploy.yml

```
@@ -73,6 +73,34 @@ phases:
+        if [ "$CODEBUILD_BUILD_SUCCEEDING" != "1" ]; then
+          # Get GitHub token for authentication
+          SECRET_ID=$(aws secretsmanager list-secrets --query "SecretList[?starts_with(Name, 'github-token-') && DeletedDate==null].Name" --output text)
+          if [ -z "$SECRET_ID" ]; then
+            echo "Error: No active GitHub token secret found for post-build comment."
+          else
+            aws secretsmanager get-secret-value --secret-id "$SECRET_ID" --query 'SecretString' --output text | jq -r '.token' | gh auth login --with-token
+
+            COMMENT_BODY="❌ Sandbox Deployment Failed
+            The deployment to the sandbox environment has failed.
+            Please review the CodeBuild logs and try again. 🔧"
+            
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)